mobile/src/main/res/values/strings.xml

```diff
@@ -116,6 +116,10 @@
     <string name="settings_server_primary_url" translatable="false">https://www.openhab.org/docs/apps/android.html#multi-server-support</string>
     <string name="settings_server_default_name">openHAB %d</string>
     <string name="click_here_for_more_information">Touch here for more information</string>
+    <string name="settings_multi_server_wifi_ssid_summary">The app will select this server if it\'s connected to this Wi-Fi name</string>
```

Comment:
Consider on/off strings:
`The app will select this server when connected to Wi-Fi '%s'`
`The app will not automatically select this server based on connected Wi-Fi network`

Reply:
Done